.github/workflows/packages_creation.yml

```diff
@@ -124,9 +139,11 @@ jobs:
     - name: install deps
       shell: bash
       run: |
-        apt-get update
-        apt-get install -y git
-        apt-get install -y lsb-release
+        apt update
+        apt install -y git
+        apt install -y lsb-release
+        apt install -y build-essential wget
+        apt install -y libsqlite3-dev qml-module-qt-labs-folderlistmodel qml-module-qt-labs-settings libopenni2-dev libi2c-dev libv4l-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-libav gstreamer1.0-tools libgraphviz-dev libqcustomplot-dev
```

Comment:
Why is this necessary? The .deb package is happening in the chroot environment, and so the packages should be installed there, not in the host environment, right?

Reply:
I am not using chroot, just using the host environment. If I am not missing something I think that using this infrastructure the chroot is not needed anymore, but correct me if I am wrong

Reply:
If you are not using the `chroot`, and you are just running the job on `ubuntu-latest` (that I am not really sure if it is 18.04 or 20.04, but it is not relevant) how can you generate packages for different Ubuntu distributions?

Reply:
Sorry, I was confused, `preliminary-steps` run `ubuntu-latest`, then there are ubuntu-18.04 and ubuntu-20.04 jobs for `normal-build` and debian buster using `docker-build` . Probably it is unrelated this, but why can't we  build everything with `docker-build` and specifying the docker images for Ubuntu  18.04 and 20.04 ?

Reply:
> can't we build everything with docker-build and specifying the docker images for Ubuntu 18.04 and 20.04?

Yes sure, I can do it! It would simplify the yml.
For now, I have a problem using the `upload-asset` action, the one I used worked, I don't understand why the standard one does not

Reply:
Perfect! For me if this version is working we can also merge it as it is, I just want to understand if there was a specific reason for this duplication, but we can cleanup it in the future.